src/parser.c

```diff
@@ -400,62 +478,63 @@ struct l_queue *parser_schema_to_list(const char *json_str)
 	 */
 
 	for (i = 0; i < json_object_array_length(jobjarray); i++) {
-
+		has_err = true;
```

Comment:
Why the variable has_err is not being used in any verification?

Reply:
This variable is not used indeed. I will remove it